src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx

```diff
@@ -45,21 +46,30 @@ import javascript from '@cf/images/icons/javascript.svg'
 import python from '@cf/images/icons/python.svg'
 import css from './AddKeyDialog.module.scss'
 
-interface Props {
+export interface AddKeyDialogProps {
   disabled?: boolean
   primary?: boolean
   environment: EnvironmentResponseDTO
   onCreate: (newKey: ApiKey, hideModal: () => void) => void
   buttonProps?: ButtonProps
   keyType?: EnvironmentSDKKeyType
+  apiKeys?: ApiKey[]
 }
 
 interface KeyValues {
   name: string
   type: EnvironmentSDKKeyType
 }
 
-const AddKeyDialog: React.FC<Props> = ({ disabled, primary, environment, onCreate, buttonProps, keyType }) => {
+const AddKeyDialog: React.FC<AddKeyDialogProps> = ({
+  disabled,
+  primary,
+  environment,
+  onCreate,
+  buttonProps,
+  keyType,
+  apiKeys
```

Comment:
You can simplify your code later by defaulting to a blank array here
```suggestion
  apiKeys = []
```

Then you can drop the optional chaining later on.

Reply:
done